src/pages/events/EventCard.css

```diff
@@ -0,0 +1,31 @@
+.event-card {
```

Comment:
* decrease card size 
* spacing between cards
* permanent shadow (not on hover) (take feedback from others In group on this)

Reply:
Replace Description with some-kind of text-buttons
1. Register
2. Details